pylmod/membership.py

```diff
@@ -18,7 +19,106 @@ def __init__(
             self,
             cert,
             urlbase='https://learning-modules.mit.edu:8443/',
+            cuuid=None
     ):
         super(Membership, self).__init__(cert, urlbase)
         # Add service base
         self.urlbase += 'service/membership/'
+        if cuuid is not None:
+            self.course_id = self.get_course_id(cuuid)
+
+    def get_staff(self, course_id='', simple=False):
+        """Get the staff roster for a course.
+
+        Get a list of staff members for a given course,
+        specified by a course id.
+
+        Args:
+            course_id (int): unique identifier for course, i.e. ``2314``
+
+        Raises:
+            requests.RequestException: Exception connection error
+            ValueError: Unable to decode response content
+
+        Returns:
+            list: list of dictionaries containing staff data
+
+            An example return value is:
+
+            .. code-block:: python
+
+                {
+                    u'response':
+                    {
+                        u'docs':
```

Comment:
what is this madness?  Also, I would think simple would just return the displayNames per the other `simple` conventions.  Also, is there ever any other key besides `docs`?  If not, we should return the contents of docs, since the other data is meta data, if not, we should probably do an unravel and set an attribute with each staff entry that is whatever data "docs" is supposed to represent so the structure is flat.